src/GeneralStateTestsFiller/stPreCompiledContracts2/CallEcrecoverInvalidSignatureFiller.json

```diff
@@ -0,0 +1,60 @@
+{
+    "CallEcrecoverInvalidSignature" : {
+        "env" : {
```

Comment:
could you please add 
```
"_info" : {
     "comment" : " Test Description "
}
```

section describing the logic behind LLL code. please ?

Reply:
I didn't know that was possible but definitely good to have.  I've added some, let me know if they explain things well enough.